src/cmd-meta

```diff
@@ -7,14 +7,20 @@ import argparse
 import os
 import sys
 
-sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
+COSA_PATH = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, COSA_PATH)
 from cosalib.meta import GenericBuildMeta as Meta
 
 
 def new_cli():
     parser = argparse.ArgumentParser()
     parser.add_argument('--workdir', default=os.getcwd())
     parser.add_argument('--build', default='latest')
+    parser.add_argument('--allow-invalid', '--skip-validation',
```

Comment:
I feel like those should be different switches. `--skip-validation` skips schema validation entirely, whereas `--allow-invalid` would try to validate, but not error out if it failed (printing the error on stderr).

Reply:
I dropped `--allow-invalid` and went for `--skip-validation`.